File: ImportService/ImportDataManager/DataStandardSystem/DataStandardSystem/Data/Model/Doserate.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataStandardSystem.Data
{
    public class Doserate
    {
        private String apiOffer;
        private String apiKey;
        private String apiName;
        private DateTime apiOfferTime;
        private DateTime dbUpdateTime;
        private Double doserateValue;
        private String unit;

        public String ApiOffer
        {
            get { return apiOffer; }
            set { apiOffer = value; }
        }

        public String ApiKey
        {
            get { return apiKey; }
            set { apiKey = value; }
        }

        public String ApiName
        {
            get { return apiName; }
            set { apiName = value; }
        }

        public DateTime ApiOfferTime
        {
            get { return apiOfferTime; }
            set { apiOfferTime = value; }
        }

        public DateTime DbUpdateTime
        {
            get { return dbUpdateTime; }
            set { dbUpdateTime = value; }
        }

        public Double DoserateValue
        {
            get { return doserateValue; }
            set { doserateValue = value; }
        }

        public String Unit
        {
            get { return unit; }
            set { unit = value; }
        }
    }
}

File: ImportService/ImportDataAPIService/src/main/java/com/service/Model/Doserate.java
package com.service.Model;

import org.springframework.data.mongodb.core.mapping.Document;

import java.util.ArrayList;

/**
 * Created by user on 2019-04-03.
 */

@Document(collection = "doserate")
public class Doserate {

    public String apiOffer;
    public String apiKey;
    public String apiName;
    public double doserateValue;
    public String unit;
    public String apiOfferTime;
    public String dbUpdateTime;

    public ArrayList<Doserate> dataList;

    public String getApiOffer() {
        return apiOffer;
    }

    public void setApiOffer(String apiOffer) {
        this.apiOffer = apiOffer;
    }

    public String getApiKey() {
        return apiKey;
    }

    public void setApiKey(String apiKey) {
        this.apiKey = apiKey;
    }

    public String getApiName() {
        return apiName;
    }

    public void setApiName(String apiName) {
        this.apiName = apiName;
    }

    public double getDoserateValue() {
        return doserateValue;
    }

    public void setDoserateValue(double doserateValue) {
        this.doserateValue = doserateValue;
    }

    public String getUnit() {
        return unit;
    }

    public void setUnit(String unit) {
        this.unit = unit;
    }

    public String getApiOfferTime() {
        return apiOfferTime;
    }

    public void setApiOfferTime(String apiOfferTime) {
        this.apiOfferTime = apiOfferTime;
    }

    public String getApiUpdateTime() {
        return dbUpdateTime;
    }

    public void setApiUpdateTime(String dbUpdateTime) {
        this.dbUpdateTime = dbUpdateTime;
    }

    public ArrayList<Doserate> getDataList() {
        return dataList;
    }

    public void setDataList(ArrayList<Doserate> dataList) {
        this.dataList = dataList;
    }

    @Override
    public String toString() {
        return "Doserate{" +
                "apiOffer='" + apiOffer + '\'' +
                ", apiKey='" + apiKey + '\'' +
                ", apiName='" + apiName + '\'' +
                ", doserateValue=" + doserateValue +
                ", unit='" + unit + '\'' +
                ", apiOfferTime='" + apiOfferTime + '\'' +
                ", apiUpdateTime='" + dbUpdateTime + '\'' +
                ", dataList=" + dataList +
                '}';
    }
}
